azkaban-common/src/main/java/azkaban/HadoopModule.java

```diff
@@ -52,32 +59,111 @@
   @Inject
   @Provides
   @Singleton
-  public Configuration createHadoopConfiguration() {
+  @Named("hdfsConf")
+  public Configuration createHDFSConfiguration() {
```

Comment:
This method is for hadoop related configuration, not only for HDFS storage. Will it be better to keep the original name?

Reply:
Actually it is only for HDFS storage. But thanks for bringing this up because I discovered I forgot to put the new `@Named` annotation on a `Configuration` injection in `HdfsAuth`. That is fixed now.